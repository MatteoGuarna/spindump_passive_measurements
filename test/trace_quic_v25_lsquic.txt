A test with Lsquic QUIC implementation server and Quant client.

There are four sessions:

- regular + h3 + full start
- regular + h3 + 0rtt
- regular + h3 + stateless retry
- key update + h3
- quantum-mode + h3

The commands used were:

   (rm /tmp/quant*;
   ./bin/client -3 -i en0 https://http3-test.litespeedtech.com:4433/15000 ;
   ./Bin/client -3 -i en0 https://http3-test.litespeedtech.com:4433/15000 ;
   ./Bin/client -3 -i en0 https://http3-test.litespeedtech.com:4434/15000 ;
   ./bin/client -3 -u -i en0 https://http3-test.litespeedtech.com:4433/15000 ;
   ./bin/client -3 -m -i en0 https://http3-test.litespeedtech.com:4433/15000 )


