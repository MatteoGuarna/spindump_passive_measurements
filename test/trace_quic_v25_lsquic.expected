QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 72929a82-9942ee94 (60502:4433) at 1580823783615923 new starting packets 0 0 bytes 0 0 note "No rsp,v.0xbabababa,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 72929a82-3b5b43c2119942 (60502:4433) at 1580823783709354 change starting packets 1 0 bytes 1248 0 bandwidth 1248 0 note "No rsp,v.0x00000000,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 72929a82-3b5b43c2119942 (60502:4433) at 1580823783709354 measurement starting right 93431 packets 1 0 bytes 1248 0 bandwidth 1248 0 note "No rsp,v.0x00000000,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 null-3b5b43c2119942 (60502:4433) at 1580823783711693 measurement starting left 2339 packets 1 1 bytes 1248 96 bandwidth 1248 96 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 null-0816e8995c51a3a4 (60502:4433) at 1580823783806850 change starting packets 2 1 bytes 2496 96 bandwidth 2496 96 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 null-0816e8995c51a3a4 (60502:4433) at 1580823783806850 measurement up right 95157 packets 2 1 bytes 2496 96 bandwidth 2496 96 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 635324d5-0816e8995c51a3a4 (60502:4433) at 1580823783807247 packet up responder length 1280 packets 2 3 bytes 2496 1622 bandwidth 2496 1622 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 635324d5-0816e8995c51a3a4 (60502:4433) at 1580823783807588 packet up responder length 1280 packets 2 4 bytes 2496 2902 bandwidth 2496 2902 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 635324d5-0816e8995c51a3a4 (60502:4433) at 1580823783808373 packet up initiator length 92 packets 3 4 bytes 2588 2902 bandwidth 2588 2902 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 635324d5-0816e8995c51a3a4 (60502:4433) at 1580823783808744 packet up responder length 815 packets 3 5 bytes 2588 3717 bandwidth 2588 3717 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 635324d5-0816e8995c51a3a4 (60502:4433) at 1580823783809381 packet up initiator length 91 packets 4 5 bytes 2679 3717 bandwidth 2679 3717 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 635324d5-0816e8995c51a3a4 (60502:4433) at 1580823783810729 packet up initiator length 1418 packets 5 5 bytes 4097 3717 bandwidth 4097 3717 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 635324d5-0816e8995c51a3a4 (60502:4433) at 1580823783899086 packet up responder length 92 packets 6 6 bytes 4212 3809 bandwidth 4212 3809 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 635324d5-0816e8995c51a3a4 (60502:4433) at 1580823783900463 packet up responder length 91 packets 6 7 bytes 4212 3900 bandwidth 4212 3900 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 635324d5-0816e8995c51a3a4 (60502:4433) at 1580823783904251 packet up responder length 91 packets 6 9 bytes 4212 4510 bandwidth 4212 4510 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 635324d5-0816e8995c51a3a4 (60502:4433) at 1580823783906244 measurement up full (initiator) 854 packets 7 17 bytes 4291 14750 bandwidth 4291 14750 note "V25,no R-spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 635324d5-0816e8995c51a3a4 (60502:4433) at 1580823783907735 measurement up full (initiator) 1491 packets 8 21 bytes 4374 19870 bandwidth 4374 19870 note "V25,no R-spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 635324d5-0816e8995c51a3a4 (60502:4433) at 1580823784000401 measurement up right 95011 packets 11 22 bytes 4615 20641 bandwidth 4615 20641 note "V25,spinning"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 04000760-585c77e137cd310a225fc21aebc6eb0ac1e2a4 (60503:4433) at 1580823784480142 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 04000760-08125de17703d0d9 (60503:4433) at 1580823784576390 change starting packets 1 0 bytes 1248 0 bandwidth 1248 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 04000760-08125de17703d0d9 (60503:4433) at 1580823784576390 measurement up right 96248 packets 1 0 bytes 1248 0 bandwidth 1248 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 00076016-08125de17703d0d9 (60503:4433) at 1580823784576395 packet up responder length 93 packets 1 2 bytes 1248 1014 bandwidth 1248 1014 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 00076016-08125de17703d0d9 (60503:4433) at 1580823784585778 packet up initiator length 164 packets 2 15 bytes 1412 17128 bandwidth 1412 17128 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 00076016-08125de17703d0d9 (60503:4433) at 1580823784586473 packet up initiator length 1418 packets 3 15 bytes 2830 17128 bandwidth 2830 17128 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 00076016-08125de17703d0d9 (60503:4433) at 1580823784591081 packet up responder length 953 packets 5 16 bytes 3028 18081 bandwidth 3028 18081 note "V25,0-RTT,no R-spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 00076016-08125de17703d0d9 (60503:4433) at 1580823784591497 packet up initiator length 93 packets 6 16 bytes 3121 18081 bandwidth 3121 18081 note "V25,0-RTT,no R-spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 00076016-08125de17703d0d9 (60503:4433) at 1580823784617328 measurement up full (initiator) 28390 packets 6 16 bytes 3121 18081 bandwidth 3121 18081 note "V25,0-RTT,no R-spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 00076016-08125de17703d0d9 (60503:4433) at 1580823784677499 packet up responder length 93 packets 7 17 bytes 3205 18174 bandwidth 3205 18174 note "V25,0-RTT,no R-spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 00076016-08125de17703d0d9 (60503:4433) at 1580823784679404 measurement up right 62076 packets 7 17 bytes 3205 18174 bandwidth 3205 18174 note "V25,0-RTT,spinning"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 00076016-08125de17703d0d9 (60503:4433) at 1580823784704327 measurement up left 24923 packets 7 18 bytes 3205 18256 bandwidth 3205 18256 note "V25,0-RTT,spinning"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 00076016-08125de17703d0d9 (60503:4433) at 1580823784704327 measurement up full (initiator) 86999 packets 7 18 bytes 3205 18256 bandwidth 3205 18256 note "V25,0-RTT,spinning"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 00076016-08125de17703d0d9 (60503:4433) at 1580823784706761 measurement up right 2434 packets 8 18 bytes 3292 18256 bandwidth 3292 18256 note "V25,0-RTT,spinning"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 00076016-08125de17703d0d9 (60503:4433) at 1580823784706761 measurement up full (responder) 27357 packets 8 18 bytes 3292 18256 bandwidth 3292 18256 note "V25,0-RTT,spinning"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 0462d1d1-a9f49d8ae359b2bb (60504:4434) at 1580823785244711 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 0462d1d1-0879c3e25fb7c08b (60504:4434) at 1580823785337505 change starting packets 1 0 bytes 1248 0 bandwidth 1248 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 62d1d110-0879c3e25fb7c08b (60504:4434) at 1580823785340559 measurement starting left 3054 packets 1 1 bytes 1248 145 bandwidth 1248 145 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 62d1d110-08b475a7b1bf9dad (60504:4434) at 1580823785436802 change starting packets 2 1 bytes 2496 145 bandwidth 2496 145 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 62d1d110-08b475a7b1bf9dad (60504:4434) at 1580823785436802 measurement up right 96243 packets 2 1 bytes 2496 145 bandwidth 2496 145 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 62d1d110-08b475a7b1bf9dad (60504:4434) at 1580823785436811 packet up responder length 93 packets 2 3 bytes 2496 1171 bandwidth 2496 1171 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 62d1d110-08b475a7b1bf9dad (60504:4434) at 1580823785441550 packet up initiator length 226 packets 3 16 bytes 2722 17286 bandwidth 2722 17286 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 62d1d110-08b475a7b1bf9dad (60504:4434) at 1580823785442057 packet up initiator length 1418 packets 4 16 bytes 4140 17286 bandwidth 4140 17286 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 62d1d110-08b475a7b1bf9dad (60504:4434) at 1580823785448846 packet up responder length 966 packets 6 17 bytes 4338 18252 bandwidth 4338 18252 note "V25,0-RTT,no R-spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 62d1d110-08b475a7b1bf9dad (60504:4434) at 1580823785449551 packet up initiator length 93 packets 7 17 bytes 4431 18252 bandwidth 4431 18252 note "V25,0-RTT,no R-spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 62d1d110-08b475a7b1bf9dad (60504:4434) at 1580823785476492 measurement up full (initiator) 32209 packets 7 17 bytes 4431 18252 bandwidth 4431 18252 note "V25,0-RTT,no R-spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 62d1d110-08b475a7b1bf9dad (60504:4434) at 1580823785532769 packet up responder length 92 packets 8 18 bytes 4515 18344 bandwidth 4515 18344 note "V25,0-RTT,no R-spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 62d1d110-08b475a7b1bf9dad (60504:4434) at 1580823785534774 measurement up right 58282 packets 8 18 bytes 4515 18344 bandwidth 4515 18344 note "V25,0-RTT,spinning"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 62d1d110-08b475a7b1bf9dad (60504:4434) at 1580823785560071 measurement up full (responder) 25297 packets 8 19 bytes 4515 18432 bandwidth 4515 18432 note "V25,0-RTT,spinning"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 62d1d110-08b475a7b1bf9dad (60504:4434) at 1580823785564409 measurement up left 29635 packets 8 20 bytes 4515 18509 bandwidth 4515 18509 note "V25,0-RTT,spinning"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 62d1d110-08b475a7b1bf9dad (60504:4434) at 1580823785564409 measurement up full (initiator) 87917 packets 8 20 bytes 4515 18509 bandwidth 4515 18509 note "V25,0-RTT,spinning"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 04ed935a-8efba96ffb9d28c621f0906d534e1f22d9f5 (60505:4433) at 1580823786122578 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 04ed935a-08dfd3ac0493cbce (60505:4433) at 1580823786262255 change starting packets 1 0 bytes 1248 0 bandwidth 1248 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 04ed935a-08dfd3ac0493cbce (60505:4433) at 1580823786262255 measurement up right 139677 packets 1 0 bytes 1248 0 bandwidth 1248 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 ed935a5c-08dfd3ac0493cbce (60505:4433) at 1580823786262262 packet up responder length 93 packets 1 2 bytes 1248 1014 bandwidth 1248 1014 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 ed935a5c-08dfd3ac0493cbce (60505:4433) at 1580823786263048 packet up responder length 975 packets 1 16 bytes 1248 18125 bandwidth 1248 18125 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 ed935a5c-08dfd3ac0493cbce (60505:4433) at 1580823786271937 packet up initiator length 164 packets 2 16 bytes 1412 18125 bandwidth 1412 18125 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 ed935a5c-08dfd3ac0493cbce (60505:4433) at 1580823786272730 packet up initiator length 1418 packets 3 16 bytes 2830 18125 bandwidth 2830 18125 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 ed935a5c-08dfd3ac0493cbce (60505:4433) at 1580823786275234 packet up initiator length 93 packets 5 16 bytes 3038 18125 bandwidth 3038 18125 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 ed935a5c-08dfd3ac0493cbce (60505:4433) at 1580823786362597 packet up responder length 92 packets 6 17 bytes 3123 18217 bandwidth 3123 18217 note "V25,0-RTT,no R-spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 ed935a5c-08dfd3ac0493cbce (60505:4433) at 1580823786391878 measurement up right 116363 packets 7 18 bytes 3210 18299 bandwidth 3210 18299 note "V25,0-RTT,spinning"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 ed935a5c-08dfd3ac0493cbce (60505:4433) at 1580823786391878 measurement up full (responder) 27434 packets 7 18 bytes 3210 18299 bandwidth 3210 18299 note "V25,0-RTT,spinning"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 04fdc48a-40366270939c8614bbb27876c1797773 (60506:4433) at 1580823787145901 new starting packets 0 0 bytes 0 0 note "No rsp,v25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 04fdc48a-40366270939c8614bbb27876c1797773 (60506:4433) at 1580823787145902 packet starting initiator length 851 packets 2 0 bytes 2131 0 bandwidth 2131 0 note "No rsp,v25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 04fdc48a-08184c1805ffe900 (60506:4433) at 1580823787266271 change starting packets 2 0 bytes 2131 0 bandwidth 2131 0 note "No rsp,v25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 fdc48a7c-08184c1805ffe900 (60506:4433) at 1580823787268761 measurement starting left 2490 packets 2 1 bytes 2131 153 bandwidth 2131 153 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 fdc48a7c-08184c1805ffe900 (60506:4433) at 1580823787268762 measurement starting left 2491 packets 3 1 bytes 3411 153 bandwidth 3411 153 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 fdc48a7c-0806e67f6ae3a05f (60506:4433) at 1580823787662015 change starting packets 4 1 bytes 4402 153 bandwidth 4402 153 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 fdc48a7c-0806e67f6ae3a05f (60506:4433) at 1580823787662015 measurement up right 393253 packets 4 1 bytes 4402 153 bandwidth 4402 153 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 fdc48a7c-0806e67f6ae3a05f (60506:4433) at 1580823787662021 packet up responder length 93 packets 4 3 bytes 4402 1187 bandwidth 4402 1187 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 fdc48a7c-0806e67f6ae3a05f (60506:4433) at 1580823787672425 packet up initiator length 234 packets 5 16 bytes 4636 17307 bandwidth 4636 17307 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 fdc48a7c-0806e67f6ae3a05f (60506:4433) at 1580823787673092 packet up initiator length 1418 packets 6 16 bytes 6054 17307 bandwidth 6054 17307 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 fdc48a7c-0806e67f6ae3a05f (60506:4433) at 1580823787674534 packet up responder length 979 packets 7 17 bytes 6169 18286 bandwidth 6169 18286 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 fdc48a7c-0806e67f6ae3a05f (60506:4433) at 1580823787675574 packet up initiator length 93 packets 9 17 bytes 6343 18286 bandwidth 6343 18286 note "V25,no R-spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 fdc48a7c-0806e67f6ae3a05f (60506:4433) at 1580823787763022 packet up responder length 92 packets 10 18 bytes 6427 18378 bandwidth 6427 18378 note "V25,no R-spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 fdc48a7c-0806e67f6ae3a05f (60506:4433) at 1580823787763027 measurement up right 88200 packets 10 18 bytes 6427 18378 bandwidth 6427 18378 note "V25,spinning"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 fdc48a7c-0806e67f6ae3a05f (60506:4433) at 1580823787792666 measurement up left 29639 packets 10 20 bytes 6427 18535 bandwidth 6427 18535 note "V25,spinning"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2604:a880:800:a1::1279:3001 fdc48a7c-0806e67f6ae3a05f (60506:4433) at 1580823787792666 measurement up full (initiator) 117839 packets 10 20 bytes 6427 18535 bandwidth 6427 18535 note "V25,spinning"
